mininet/cli.py

```diff
@@ -55,11 +56,10 @@ def __init__( self, mininet, stdin=sys.stdin, script=None ):
         # Local variable bindings for py command
         self.locals = { 'net': mininet }
         # Attempt to handle input
-        self.stdin = stdin
         self.inPoller = poll()
         self.inPoller.register( stdin )
         self.inputFile = script
-        Cmd.__init__( self )
+        Cmd.__init__( self, stdin=stdin, *args, **kwargs )
```

Comment:
I don't think Python allows you to have named parameters before positional parameters.

Reply:
I modified the call